src/quo/components/list_items/network_list/view.cljs

```diff
@@ -0,0 +1,75 @@
+(ns quo.components.list-items.network-list.view
+  (:require
+    [quo.components.list-items.network-list.style :as style]
+    [quo.components.markdown.text :as text]
+    [quo.foundations.resources :as quo.resources]
+    [quo.theme :as quo.theme]
+    [react-native.core :as rn]
+    [reagent.core :as reagent]
+    [schema.core :as schema]))
+
+(defn- info
+  [{:keys [network-name label]}]
+  [rn/view {:style style/info}
+   [rn/image
+    {:source (quo.resources/get-network network-name)
+     :style  style/network-image}]
+   [rn/view
+    [text/text
+     {:weight          :semi-bold
+      :style           {:text-transform :capitalize}
+      :number-of-lines 1}
+     (if-not (empty? label) label "-")]]])
```

Comment:
is `label` a string?
If so, maybe we want to use `(not string/blank?)`. if not, then maybe it's better to use `seq` instead of `not` + `empty?`.


from Clojure docs:
![image](https://github.com/status-im/status-mobile/assets/90291778/e7835156-f022-4481-9743-230705aa0a3e)

Reply:
thanks for the feedback @ulisesmac. `label` is a `string`, have updated to use `string/blank?`